win/WIN_ButtonUtils.cpp

```diff
@@ -0,0 +1,50 @@
+#include "WIN_ButtonUtils.h"
+#include <SDL.h>
+#include <SDL_image.h>
+#include <cstdio>
+#include <cassert>
+
+using namespace win;
+
+SDL_Texture* ButtonUtils::loadSprite(SDL_Renderer* renderer, const char* path)
+{
+	//The final texture
+	SDL_Texture* newTexture = nullptr;
+
+	//Load image at specified path
+	auto* loadedSurface = IMG_Load(path);
+	if (loadedSurface == nullptr) {
+		printf("Unable to load image %s! SDL_image Error: %s\n", path, IMG_GetError());
```

Comment:
So this we should talk about, because print messages are too easy to miss / ignore. Handling errors is a fiddly but important topic and I have strong opinions about how to do it :-) Will schedule a meeting for today.

Reply:
done